components/dropdown/dropdown-content-mixin.js

```diff
@@ -0,0 +1,556 @@
+import { clearDismissible, setDismissible } from '../../helpers/dismissible.js';
+import { findComposedAncestor, isComposedAncestor } from '../../helpers/dom.js';
+import { getComposedActiveElement, getFirstFocusableDescendant, getPreviousFocusableAncestor } from '../../helpers/focus.js';
+import { html } from 'lit-element/lit-element.js';
+import { styleMap } from 'lit-html/directives/style-map.js';
+
+export const DropdownContentMixin = superclass => class extends superclass {

```

Comment:
Will need to import and extend the `RtlMixin` here (as per other comment in the styles file).